### PR TITLE
Uniprotkb edismax qf

### PR DESCRIPTION
**Pull request by eddturner**

### CODE REVIEW

File: indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java [hunk 1]
```diff
@@ -74,29 +74,15 @@ void convertModelOrganism() {
                         "Homo sapiens",
                         "Human",
                         "Homo sapian",
-                        "HUMAN",
+                        //                        "HUMAN",
```
Comment (LeonardoGonzales) [inline indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java:77]: I think we will need the mnemonic for organism search. For taxonomy (lineage) we do not need.. (we can double check it with Sangya)

**Review by LeonardoGonzales — COMMENTED** [review]
Excellent Job!!!

File: indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java (hunk 1 above)
Comment (eddturner) [reply indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java:77]: Jie said we didn't need this one, and uniprot.org doesn't do it either.

for: https://www.ebi.ac.uk/proteins/api/taxonomy/id/55

https://www.uniprot.org/uniprot/?query=taxonomy%3A%22Polyangium+%5B55%5D%22&sort=score (can search by scientific name)
https://www.uniprot.org/uniprot/?query=9DELT&sort=score (mnemonic hits the second part of the entry ID)
https://www.uniprot.org/uniprot/?query=taxonomy%3A9delt&sort=score (mnemonic not indexed)

... waiting for a chat with sangya to check if this is fine to follow.

Comment (eddturner) [reply indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java:77]: Confirmed not needed by Sangya

File: index-config/src/main/solr-config/uniprot-collections/uniprot/conf/schema.xml [hunk 2]
```diff
@@ -79,13 +76,10 @@
 		stored="false" required="false" multiValued="true" />
 
 	<field name="accession" type="basic_ci" indexed="true"
-		stored="true" required="true" multiValued="true" />
+		stored="false" required="true" multiValued="true" />
 	<copyField source="sec_acc" dest="accession" />
 	<copyField source="accession_id" dest="accession" />
 
-	<!-- go used the solr default search, it does not have (partOf,parent) information -->
```
[the remark this answers is not in the document]
Comment (eddturner) [reply index-config/src/main/solr-config/uniprot-collections/uniprot/conf/schema.xml:86]: forgot to say i made the changes we talked about yesterday. i.e., goes and goId have ancestor info in them, but are not part of default search. content field (part of default search) has no ancestor info in it. and there is a test for this.

**Pull request merged**